src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrType.cs

```diff
@@ -75,10 +81,25 @@ internal DkmClrType MakeArrayType()
                 _lmrType.MakeArrayType());
         }
 
-        internal object Instantiate(params object[] args)
+        internal DkmClrValue Instantiate(params object[] args)
+        {
+            return Instantiate(args, null, DkmEvaluationResultFlags.None);
+        }
+
+        internal DkmClrValue Instantiate(
```

Comment:
I couldn't find what calls this overload, except for the method above. Could this be private?